src/System.Net.WebSockets/src/System.Net.WebSockets.csproj

```diff
@@ -44,4 +45,7 @@
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.Threading.Timer" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'!='netcoreapp'">
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces" />
+  </ItemGroup>
```

Comment:
None of these changes should be necessary.  This project doesn't build for netfx or netstandard.
https://github.com/dotnet/corefx/blob/74a13ffba6d662decb073f2851a98435b80a65a2/src/System.Net.WebSockets/src/Configurations.props#L4-L5